contrib/pkg/output/errhandlers/err_handlers.go

```diff
@@ -9,23 +9,33 @@ import (
 
 // this file contains ErrorHandlers for handling errors created when writing output snapshots
 
+func ResourceWriteError(resource ezkube.Object, err error) error {
+	return eris.Wrapf(err, "writing resource %v failed", sets.Key(resource))
+}
+func ResourceDeleteError(resource ezkube.Object, err error) error {
+	return eris.Wrapf(err, "deleting resource %v failed", sets.Key(resource))
+}
+func ListError(err error) error {
+	return eris.Wrapf(err, "listing failed")
+}
+
 type AppendingErrHandler struct {
 	errs error
 }
 
-func (a AppendingErrHandler) HandleWriteError(resource ezkube.Object, err error) {
-	a.errs = multierror.Append(a.errs, eris.Wrapf(err, "writing resource %v failed", sets.Key(resource)))
+func (a *AppendingErrHandler) HandleWriteError(resource ezkube.Object, err error) {
+	a.errs = multierror.Append(a.errs, ResourceWriteError(resource, err))
 }
 
-func (a AppendingErrHandler) HandleDeleteError(resource ezkube.Object, err error) {
-	a.errs = multierror.Append(a.errs, eris.Wrapf(err, "deleting resource %v failed", sets.Key(resource)))
+func (a *AppendingErrHandler) HandleDeleteError(resource ezkube.Object, err error) {
+	a.errs = multierror.Append(a.errs, ResourceDeleteError(resource, err))
 }
 
-func (a AppendingErrHandler) HandleListError(err error) {
-	a.errs = multierror.Append(a.errs, eris.Wrapf(err, "listing failed"))
+func (a *AppendingErrHandler) HandleListError(err error) {
+	a.errs = multierror.Append(a.errs, ListError(err))
 }
 
 // returns the errors collected by the handler
```

Comment:
Nit: was already like this, but since we are here
```suggestion
// Errors returns the errors collected by the handler
```